src/components/FileList/FileList.jsx

```diff
@@ -8,7 +8,7 @@ const FileList = (props) => {
   console.log(files);
 
   const content = files.map((file) => (
-    <FileItem
+    <FileItem className = {styles.fileslist}
```

Comment:
You are using fileslist styling function but there is not fileslist class associated in the corresponding css file.